jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -2,12 +2,17 @@
 set -x
 set -e
 
+
 # Stop host mongo and rabbitmq service
 # otherwise mongo and rabbitmq service won't run normally
 # use host mongo and rabbitmq is a chioce but the coverage may be decreased.
 # services will be restart in cleanup.sh, which script will always be executed.
-echo $SUDO_PASSWORD |sudo -S service mongodb stop
-echo $SUDO_PASSWORD |sudo -S service rabbitmq-server stop
+if [ "$(service mongodb         status|grep  start )" != "" ]; then
+    echo $SUDO_PASSWORD |sudo -S service mongodb stop
+fi
+if [ "$(service rabbitmq-server status|grep  start )" != "" ]; then
```

Comment:
Good catch @PengTian0 !

I found the problem only exists for mongodb. in another word "service mongodb stop" will return 1 when mongodb already stopped.

but rabbitmq-server does a better work.  duplicated stop will do no harm.
```
jenkins@vmslave:~$ sudo service rabbitmq-server stop
sudo: unable to resolve host vmslave
 * Stopping message broker rabbitmq-server                                      ech * message broker already stopped
                                                                         [ OK ]
jenkins@vmslave:~$ echo $?
0
```